cmd/version/github/client.go

```diff
@@ -0,0 +1,23 @@
+package github
+
+import (
+	"context"
+	"github.com/google/go-github/github"
+)
+
+type LatestReleaseFetcher interface {
+	LatestRelease(owner, repository string) (*github.RepositoryRelease, error)
+}
+
+type client struct {
+	client *github.Client
+}
+
+func NewClient() *client {
+	return &client{github.NewClient(nil)}
+}
+
+func (gc *client) LatestRelease(owner, repository string) (*github.RepositoryRelease, error) {
```

Comment:
Instead of returning '*github.RepositoryRelease' in the LatestRelease function, you can return string directly. Then in the caller you can directly use the value instead of '*release.TagName' in version.go

Reply:
Done